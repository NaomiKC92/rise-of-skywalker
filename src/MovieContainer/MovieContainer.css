* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}


.grid-2 {
  background-attachment: fixed;
  background-image: url('../images/fleet-img.png');
  background-repeat: no-repeat;
  background-size: cover;
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  justify-content: center;
  padding: 50px;
}
