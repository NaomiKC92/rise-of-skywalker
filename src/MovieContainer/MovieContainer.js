import React from 'react';
import MovieCard from '../MovieCard/MovieCard';

const MovieContainer = props => {
  const { movies } = props;

  const allMovies = movies.map((movie, i) => {
    return <MovieCard title={movie.title} key={i} />;
  });

  return (
    <div>
      <h1>{allMovies}</h1>
    </div>
  );
};

export default MovieContainer;
