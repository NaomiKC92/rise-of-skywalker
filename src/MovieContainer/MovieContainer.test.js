import React from 'react';
import { shallow } from 'enzyme';
import MovieContainer from './MovieContainer';

describe('MOVIE CONTAINER', () => {
  let wrapper;

  const mockMovies = [];
  const mockSetupCharacters = jest.fn();
  const mockUpdateCrawling = jest.fn();

  beforeEach(() => {
    wrapper = shallow(
      <MovieContainer
        movies={mockMovies}
        setupCharacters={mockSetupCharacters}
        updateCrawling={mockUpdateCrawling}
      />
    );
  });

  it('should match the MovieContainer Component Snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
