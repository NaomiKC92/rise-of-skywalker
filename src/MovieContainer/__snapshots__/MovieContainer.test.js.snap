// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MOVIE CONTAINER should match the MovieContainer Component Snapshot 1`] = `
<div
  className="grid-2"
/>
`;
